packages/api/lib/utils.js

```diff
@@ -71,10 +72,10 @@ function extractDate(payload, dateField) {
  * @returns {Integer} - sum of granule file sizes in bytes
  */
 function getGranuleProductVolume(granuleFiles) {
-  const fileSizes = granuleFiles.map((file) => file.fileSize)
-    .filter((size) => size);
-
-  return fileSizes.reduce((a, b) => a + b);
+  return granuleFiles
```

Comment:
The changes made in this file are a refactor and do not contain functionality changes.